platform-api/src/models/platform.py

```
@@ -0,0 +1,19 @@
+from pydantic import BaseModel
+from typing import List, Dict, Any, Optional
```

Ruff (0.15.11)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)